content/en/real_user_monitoring/platform/connect_rum_and_traces.md

```diff
@@ -524,6 +524,9 @@ Datadog uses the distributed tracing protocol and sets up the following HTTP hea
 : To make sure that the Agent keeps the trace.
 {{% /tab %}}
 {{% tab "W3C Trace Context" %}}
+
+The W3C Trace Context header is sent by default. This means you need to configure `traceparent` for CORS servers in addition to the default Datadog-specific headers.
```

Comment:
with the default behaviour clarified above and with the CORS requirements already explained below, this paragraph seems superfluous to me.
am I missing something?